eta/core/web.py

```diff
@@ -168,8 +169,13 @@ def _do_download(self, r, f):
         size_bits = 8 * size_bytes if size_bytes is not None else None
         total_downloaded_bytes = 0
 
+        quiet = self.quiet
```

Comment:
I do want to note that in the motivating case here of the [fiftyone.utils.openimages](https://github.com/voxel51/fiftyone/blob/develop/fiftyone/utils/openimages.py) module, some effort was already put in to show progress bar/logging messages in a bespoke and hopefully helpful way:
- `quiet=-1`: shows a progress bar when downloading the image IDs for (only) the `"train"` split (since it is much larger): https://github.com/voxel51/fiftyone/blob/a5d2209903d3699be3d310a55ab1e518f858050a/fiftyone/utils/openimages.py#L1864
- `quiet=1`: this one suppresses not only the progress bar but also the logging message indicating that a file is being downloaded (only) for the `"validation"` split (I cannot remember why though!?): https://github.com/voxel51/fiftyone/blob/a5d2209903d3699be3d310a55ab1e518f858050a/fiftyone/utils/openimages.py#L1908

Maybe there's a way to improve user experience just by tweaking the `quiet` parameters over there?

Reply:
I think that makes sense; I was thinking about a case where a user wants to turn off all progress bars but couldn't figure out the exact logic.

I like that we can turn `quiet` into a more expressive parameter that is also backward compatible:
- true/false (bool): if quiet is true, do not show any progress bar. if quiet is false, show all progress bars. This is the current behavior.
- "best" (str): only shows the progress bar if the file size exceeds certain internal values.
- (int): only show the progress bar if the file size exceeds the value passed in.